conf/application.conf

```diff
@@ -111,12 +111,10 @@ geotrellis.catalog = "app/assets/catalog.json"
 environment-type = "test"
 environment-type = ${?ENV_TYPE}
 
-smtp.host="smtp.gmail.com"
-smtp.port=465
-smtp.ssl=true
```

Comment:
Do we need to turn off ssl? Will it not work with ssl? Do we need to do some additional configuration?

Reply:
SendinBlue uses TSL encryption:
https://help.sendinblue.com/hc/en-us/articles/115000202824-Email-encryption#:~:text=for%20incoming%20connections%2C%20we%20recommend,protocol%20whenever%20it%20is%20available.
From my understanding, this means we don't use SSL...? I'm not exactly sure. It worked tho! What do you think?

Reply:
Okay, I think we might need to mark `smtp.tls=true` then. Can you test with that and see if everything still works? And if you notice any changes from that?

Reply:
I added `smtp.tls=true` and nothing changed (it still works). Do you think that it's worth pushing?

Reply:
Yeah I think it's worth it. Thanks @shokiami